notebooks/pudl-catalog.ipynb

```diff
@@ -6,7 +6,27 @@
     "tags": []
```

Comment:
<p>In this notebook is this the first cell where the data is accessed and the cache is created? Or does this cell not really "access the data". It might be good to clarify if data is being downloaded here and if this is the first time the requester pays bucket is touched and the requester is billed? Also, in the case where the user doesn't have requester pays correctly set up this would be the first time they get an error? It could be good to point out that if this cell doesn't run correctly then likely GCP isn't set up correctly.</p><p> </p><p>In the above cell where the CEMS data is inspected, does that not download data because it just references a data source not the actual database?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl-catalog/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Hmm, I don't <strong>think</strong> you should be accessing the data at this point. This is just looking at catalog level metadata.  </p><p> </p>

Reply:
<p>Is this the step where it seemed to download for you? I guess this is where it's creating the SQLiteCatalog object.</p>

Reply:
I deleted the cache and reran and this cell took me 27 seconds. Later, when the ferc1 subcatalog is accessed for the first time to get the container, I think the rest downloads because that cell took me another 20 seconds. Does that make any sense?